README.md

```diff
@@ -81,10 +113,9 @@ changes from feature branches into your `main` branch.
   - [CSS Box Model](http://www.w3schools.com/css/css_boxmodel.asp)
   - [CSS Float](http://www.w3schools.com/css/css_float.asp)- How to position elements next to each other
   - [CSS Display and Visibility](http://www.w3schools.com/css/css_display_visibility.asp)
```

Comment:
I'm not sure this one in particular is that useful for this lab.

Reply:
Float, you mean?

Yeah, I see what you're saying. I feel like floats are only really useful if you're trying to wrap text around an image or something. 🤷🏻‍♂️

Reply:
I did mean [CSS Display and Visibility](http://www.w3schools.com/css/css_display_visibility.asp) but maybe it is important to show the difference between block and inline?

Reply:
I think that all these CSS links have been added over the years in response to some specific student questions or confusions. The problem is that each person/group tends to go different directions and have different issues they wrestle with.

You don't want the list of resources to get infinitely long, reflecting a winding, complex past. On the other hand you don't want to remove things that have a high likelihood of being useful. I think that the difference between block and inline has come up numerous times, so I'm inclined to leave that bit in.

Reply:
Makes sense, thanks.

(FWIW, I'd also be inclined to keep Block vs. Inline—it's a decently big deal. 🙂)